_docs/geyser/anticheat-compatibility.md

```diff
@@ -14,14 +14,15 @@ Full Compatibility (Checks Bedrock Players Accurately)
 - [Themis](https://www.spigotmc.org/resources/90766/){:target="_blank"} - Last Updated on 15th Jun 2022
 - [Negativity V2](https://www.spigotmc.org/resources/86874/){:target="_blank"} (Paid) - Last Updated on 21st March 2022
 - [SoaromaSAC](https://www.spigotmc.org/resources/87702/){:target="_blank"} - Last Updated on 27th March 2022
+- [Spartan](https://www.spigotmc.org/resources/25638/){:target="_blank"} (Paid) - Last Updated on 3rd January 2023
+
 
 Partially Compatible (Does not Check/Ignores Bedrock Players)
 
 - [AAC](https://www.spigotmc.org/resources/6442/){:target="_blank"} (Paid) - Last Updated on 19th August 2021
 - [Flappy Anticheat](https://www.spigotmc.org/resources/92180/){:target="_blank"} - Last Updated on 14th September 2021
-- [GrimAC](https://github.com/MWHunter/Grim){:target="_blank"} - Last Updated on 24th March 2022
+- [GrimAC](https://www.spigotmc.org/resources/99923/){:target="_blank"} (Unsupported) - Last Updated on 3rd January 2023
```

Comment:
Is it really unsupported? see the GH link sent by @hasankayra04 ; seems to be maintained

Reply:
Also i would remove "(Unsupported)" and update last updated date.